Task.3/main.js

```diff
@@ -0,0 +1,20 @@
+let userName = prompt("Ваше ім'я?");
+let userSurname = prompt("Ваше Прізвище?");
+let userFavoreNumber = prompt("Ваше улюблене число?");
+alert(`Вітаю ${userName} ${userSurname}! Ваше улюблене число - ${userFavoreNumber}`);
+
+let firstNumber = +prompt("Введіть перше число");
+let secondNumber = +prompt("Введіть друге число");
+
+console.log(firstNumber + secondNumber, firstNumber - secondNumber, firstNumber * secondNumber, firstNumber / secondNumber);
+
+userBirthYear = prompt("Рік Вашого народження?");
+yearNow = new Date().getFullYear();
+userAge = yearNow - userBirthYear;
+console.log(userAge);
+
+console.log(`Остача від ділення числа ${firstNumber} на число ${secondNumber} рівна ${firstNumber / secondNumber}`);
```

Comment:
Here, you've used the wrong operator. You need to use `%`

```suggestion
console.log(`Остача від ділення числа ${firstNumber} на число ${secondNumber} рівна ${firstNumber % secondNumber}`);
```